app/assets/stylesheets/application/homepage/_event_thumbnail.scss

```diff
@@ -0,0 +1,55 @@
+.event-thumbnail {
+  background-color: $secondary-background-color;
+  vertical-align: top;
+  text-align: left;
+  position: relative;
+  display: inline-block;
+  margin: 20px 7px;
+  width: 292px;
+
+  &:hover {
+    background-color: #ece9e7;
+  }
+
+  a {
+    text-decoration: none;
+    color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.